sources/packages/web/src/components/aest/students/modals/BypassRestrictionModal.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <v-form ref="bypassRestrictionForm">
+    <modal-dialog-base :showDialog="showDialog" title="Bypass restriction">
+      >
+      <template #content>
+        <h3 class="category-header-medium my-4">Bypass information</h3>
+        <error-summary :errors="bypassRestrictionForm.errors" />
+        <v-card class="pa-4 border">
+          <v-alert
+            variant="outlined"
+            icon="fa:fa fa-triangle-exclamation"
+            class="sims-banner v-alert text-warning my-3"
+            v-if="
+              !applicationRestrictionBypassId &&
+              restrictionsToBypass.length === 0
+            "
+          >
+            No active restrictions available to be bypassed or all active
+            restrictions already have an active bypass.
+          </v-alert>
+          <span class="label-bold"
+            >I want this application to bypass the following restriction:</span
+          >
+          <v-select
+            density="compact"
+            :items="restrictionsToBypass"
+            v-model="formModel.studentRestrictionId"
+            variant="outlined"
+            :rules="[(v) => checkNullOrEmptyRule(v, 'Restriction')]"
+            :disabled="!!applicationRestrictionBypassId"
+          />
+          <span class="label-bold">Until</span>
+          <v-radio-group
+            inline
+            v-model="formModel.bypassBehavior"
+            color="primary"
+            class="mt-2"
+            :rules="[(v) => checkNullOrEmptyRule(v, 'Until')]"
+            :disabled="!!applicationRestrictionBypassId"
+          >
+            <v-radio
+              label="The next scheduled disbursement has been issued. Note: If the application is reassessed, the next disbursement will be ignored."
+              value="Next disbursement only"
+              color="primary"
+            ></v-radio>
+            <v-radio
+              label="All disbursements associated with this application have been issued."
+              value="All disbursements"
+              color="primary"
+            ></v-radio>
+          </v-radio-group>
+          <span class="label-bold">Notes</span>
+          <v-textarea
+            label="Notes"
+            variant="outlined"
+            :hide-details="true"
+            v-model="formModel.note"
+            :rules="[checkNotesLengthRule]"
+            required
+            v-if="!applicationRestrictionBypassId"
+          />
+          <template v-if="restrictionBypassDetails.createdDate">
+            <p class="pt-1 brand-gray-text">
+              {{ restrictionBypassDetails.creationNote }}
+            </p>
+            <v-row>
+              <v-col class="mt-2">
+                <div class="label-bold">Date created:</div>
+                {{ dateOnlyLongString(restrictionBypassDetails.createdDate) }}
+              </v-col>
```

Comment:
Please check the usage of existing components like the one below.
https://github.com/bcgov/SIMS/blob/b5b068683167801012496bfa260d2a96742e3bc1/sources/packages/web/src/components/common/restriction/ViewRestriction.vue#L22-L26